packages/@lightningjs/ui-components/src/components/Radio/Radio.js

```diff
@@ -93,7 +88,7 @@ export default class Radio extends Base {
       texture: lng.Tools.getRoundRect(
         this.w - 2,
         this.h - 2,
-        this.style.radius,
+        getMaxRoundRadius(this.style.radius, this.w, this.h, 0),
```

Comment:
same as the above comment about Checkbox. If I set the radius in the component styles to 15, I get odd behavior.